pkgs/development/ruby-modules/gem-config/default.nix

```diff
@@ -129,6 +129,12 @@ in
     ];
   };
 
+  pg_query = attrs: {
+    postPatch = ''
+      ln -s ${libpg_query}/lib ${libpg_query.name}
```

Comment:
This doesn’t work… And is pretty awful, but take a look at https://github.com/lfittl/pg_query/blob/master/ext/pg_query/extconf.rb#L8-L11

What do, @Mic92? :-(